integration/spark/src/main/scala/org/apache/spark/sql/execution/command/view/CarbonCreateMVCommand.scala

```diff
@@ -192,9 +192,10 @@ case class CarbonCreateMVCommand(
           throw new MalformedCarbonCommandException(
             "Cannot create mv on stream table " + table.getTableUniqueName)
         }
-        if (SegmentStatusManager.isLoadInProgressInTable(table)) {
+        if (SegmentStatusManager.isOverwriteInProgressInTable(table)) {
```

Comment:
why change it?

Reply:
Create MV While load (LOAD or INSERT INTO) is in progress could also be allowed, I think, and there is corresponding test to justify it in test class "TestCreateMVWithTimeSeries", so I changed the method. What's your opinion aboout it?